kie-api/src/main/java/org/kie/api/pmml/PMML4Result.java

```diff
@@ -192,7 +196,7 @@ public Object getResultValue(String objName, String objField, Object...params) {
                         }
                     }
                 } catch (SecurityException | IllegalAccessException | IllegalArgumentException | InvocationTargetException e1) {
-                    e1.printStackTrace();
+                    LOG.error("Exception", e1);
```

Comment:
to be noticed logger per usual is declared static so shouldn't influence on JAXB marshalling

Reply:
:+1: